lib/mongo/libmongocrypt.rb

```diff
@@ -13,7 +13,10 @@
 # limitations under the License.
 
 module Mongo
-  class Libmongocrypt
+  module Libmongocrypt
     autoload(:Binding, 'mongo/libmongocrypt/binding.rb')
   end
 end
+
+require 'mongo/libmongocrypt/binary.rb'
+require 'mongo/libmongocrypt/mongocrypt_error.rb'
```

Comment:
I'm wondering if it would make more sense to autoload these as well so that FFI doesn't need to be loaded until it's actually going to be used (see how I included ffi in the enterprise auth and kerberos gemfiles above). Open to opinions.

Reply:
I think it's a fine idea, and I believe the `.rb` suffix is unnecessary in the requires.